example/googleMapsExample.js

```diff
@@ -0,0 +1,465 @@
+	displayOptions.open();
+
+	const mapsOptions = gui.addFolder( 'GMaps' );
+	mapsOptions.add( params, 'apiKey' );
+	mapsOptions.add( params, 'reload' );
+	mapsOptions.open();
+
+	const tileOptions = gui.addFolder( 'Tiles Options' );
+	tileOptions.add( params, 'loadSiblings' );
+	tileOptions.add( params, 'stopAtEmptyTiles' );
+	tileOptions.add( params, 'errorTarget' ).min( 0 ).max( 50 );
+
+	const debug = gui.addFolder( 'Debug Options' );
+	debug.add( params, 'displayBoxBounds' );
+
+	const exampleOptions = gui.addFolder( 'Example Options' );
+	exampleOptions.add( params, 'enableUpdate' ).onChange( v => {
+
+		tiles.parseQueue.autoUpdate = v;
+		tiles.downloadQueue.autoUpdate = v;
+
+		if ( v ) {
+
+			tiles.parseQueue.scheduleJobRun();
+			tiles.downloadQueue.scheduleJobRun();
+
+		}
+
+	} );
+	exampleOptions.add( params, 'enableCacheDisplay' );
+	exampleOptions.add( params, 'enableRendererStats' );
+
+	gui.open();
+
+	statsContainer = document.createElement( 'div' );
+	document.getElementById( 'info' ).appendChild( statsContainer );
+
+	// Stats
+	stats = new Stats();
+	stats.showPanel( 0 );
+	document.body.appendChild( stats.dom );
+
+	document.addEventListener( "dblclick", event => {
+		if ( raycaster ) {
+			pointer.x = ( event.clientX / window.innerWidth ) * 2 - 1;
+			pointer.y = - ( event.clientY / window.innerHeight ) * 2 + 1;
+
+			raycaster.setFromCamera( pointer, camera );
+			const intersections = raycaster.intersectObject( tiles.group, true );
+			const numIntersections = intersections.length;
+			if ( numIntersections > 0 ) {
+				const intersection = intersections[0];
+				console.log("intersection", intersection.point.length(), intersection.distance);
+				controls.target.copy( intersection.point );
+			}
+		}
+	} );
+
+}
+
+function onWindowResize() {
+
+	camera.aspect = window.innerWidth / window.innerHeight;
+	renderer.setSize( window.innerWidth, window.innerHeight );
+
+	camera.updateProjectionMatrix();
+	renderer.setPixelRatio( window.devicePixelRatio * params.resolutionScale );
+
+}
+
+const top = { origin: new Vector3( 0, 1000000000, 0 ), dir: new Vector3( 0, -1, 0 ) };
+const side = { origin: new Vector3( 1000000000, 0, 0 ), dir: new Vector3( -1, 0, 0 ) };
+const temp = new Vector3();
+
+function animate() {
+
+	requestAnimationFrame( animate );
+
+	if ( ! tiles ) return;
+
+	if ( ! ellipsoidInitialised && raycaster ) {
+		const findSurface = ( query ) => {
+			raycaster.set( query.origin, query.dir );
+			const intersections = raycaster.intersectObject( tiles.group, true );
+			const numIntersections = intersections.length;
+			if ( numIntersections > 0 ) {
+				const intersection = intersections[0];
+				return intersection.point.length();
+			} else {
+				return undefined;
+			}
+		}
+
+		const y = findSurface( top );
+		const xz = findSurface( side );
+
+		if ( y !== undefined && xz !== undefined ) {
+			ellipsoid.radius.set( xz, y, xz );
```

Comment:
you could use the WGS constants to start from?
```
import { WGS84_RADIUS, WGS84_HEIGHT } from '../src/constants.js';
let ellipsoid = new Ellipsoid(WGS84_RADIUS, WGS84_RADIUS, WGS84_HEIGHT);
```

It's not perfect depending on the elevation of the area you're looking at, but should be good enough to initialize the camera controls if that was the main thing you're trying to do?